host/xtest/xtest_6000.c

```diff
@@ -1731,9 +1731,19 @@ static void xtest_tee_test_6018_single(ADBG_Case_t *c, uint32_t storage_id)
 	uint32_t obj;
 	uint32_t orig;
 	uint8_t block[32 * 1024];
-	size_t num_blocks = 50;
+	size_t num_blocks;
+	size_t block_size;
 	size_t n;
 
+	if (storage_id == TEE_STORAGE_PRIVATE_RPMB) {
```

Comment:
The test has to be a bit more complicated, to handle the case when `storage_id == TEE_STORAGE_PRIVATE` and the REE FS is not compiled in (hence, the second FS in order or priority is selected by the TEE core, and that is RPMB). Here is what I mean:
```C
static uint32_t fs_id_for_tee_storage_private()
{
#ifdef CFG_REE_FS
        return TEE_STORAGE_PRIVATE_REE;
#else
#ifdef CFG_RPMB_FS
        return TEE_STORAGE_PRIVATE_RPMB;
#else
#ifdef CFG_SQL_FS
        return TEE_STORAGE_PRIVATE_SQL;
#endif
#endif
#endif
}

static uint32_t real_id_for(uint32_t id)
{
        if (id == TEE_STORAGE_PRIVATE)
                return fs_id_for_tee_storage_private();
        return id;
}

static bool storage_is(uint32_t id1, uint32_t id2)
{
        return (real_id_for(id1) == real_id_for(id2));
}

static void xtest_tee_test_6018_single(ADBG_Case_t *c, uint32_t storage_id)
{
        /* ... */

        if (storage_is(storage_id, TEE_STORAGE_PRIVATE_RPMB)) {
                /* RPMB FS is a bit resource constrained */
                /* ... */
        }
```